demo/run-demo.sh

```
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# Run every CLI command documented in README.md against the demo scripts
+# and report PASS/FAIL for each. Used both as a developer smoke test
+# (`composer demo`) and as a hands-on tour of every tool.
+
+set -u
+
+cd "$(dirname "$0")/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)